content/events/16-17/fosdem.md

```diff
@@ -0,0 +1,20 @@
+---
+title: FOSDEM
+description: Zeus gaat naar FOSDEM
+created_at: 27-12-2016
+time: 04-02-2017, 05-02-2017
```

Comment:
Hmm, I haven't implemented multiple dates yet, I'll have to make an issue for this.